Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php

```diff
@@ -72,15 +73,8 @@ public function getDescription()
      */
     private function injectAlignmentPlaceholders(Tokens $tokens, $startAt = null, $endAt = null)
     {
-        if (empty($startAt)) {
-            $startAt = 0;
-        }
-
-        if (empty($endAt)) {
```

Comment:
This way there is no need to test the input for every iteration, which makes the execution a bit faster.